packages/patterns-react/src/patterns/ContentArrayWithPictograms/ContentArrayWithPictogramsItem.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import PropTypes from 'prop-types';
+import { ArrowRight20 } from '@carbon/icons-react';
+import { LinkWithIcon } from '@carbon/ibmdotcom-react';
+import * as Icons from '@carbon/pictograms-react';
```

Comment:
Doesn't this import all Pictograms?

Reply:
This was the only solution I've found in order to dynamically get them, as we can't import the component based on the prop, as far as I know.

Reply:
That could get pretty large. You should see how we at least handle it in the `ButtonGroup`, where we make the users inject it in as part of the array data as a React component and not a string.